keyboards/decent/tkl/config.h

```diff
@@ -0,0 +1,9 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define I2C1_SDA_PIN GP26
+#define I2C1_SCL_PIN GP27
+
+#define OLED_FONT_H "atari-font.c"
```

Comment:
The font existing within the default keymap, but configured as default for *all* keymaps is incorrect, and will make all "user" keymaps fail to build.

Either the font file needs to be relocated, or this configuration option moved to the keymap (which would mean all "user" keymaps use the default font).

Reply:
Font moved to the kb level